ui/raidboss/data/06-ew/hunts/ultima_thule.ts

```diff
@@ -82,29 +82,9 @@ const triggerSet: TriggerSet<Data> = {
     {
       id: 'Hunt Fan Ail Divebomb',
       type: 'StartsUsing',
-      netRegex: NetRegexes.startsUsing({ id: '6AED', source: 'Fan Ail' }),
+      netRegex: NetRegexes.startsUsing({ id: '6AED', source: 'Fan Ail', capture: false }),
       condition: (data) => data.inCombat,
-      alertText: (data, matches, output) => {
-        if (data.me === matches.target)
-          return output.divebombOnYou!();
-        return output.divebombMarker!();
-      },
-      outputStrings: {
-        divebombOnYou: {
-          en: 'Divebomb on YOU',
-          de: 'Sturzflug auf DIR',
-          fr: 'Bombe plongeante sur VOUS',
-          cn: '俯冲点名',
-          ko: '나에게 초록징',
-        },
-        divebombMarker: {
-          en: 'Away from Divebomb Marker',
-          de: 'Weg von dem Sturzflug-Marker',
-          fr: 'Éloignez-vous de la bombe plongeante',
-          cn: '躲开俯冲点名',
-          ko: '초록징 피하기',
-        },
-      },
+      response: Responses.awayFromFront(),
```

Comment:
I disagree a little here? It's away from where the divebomb marker appeared, not from the front of Fan Ail.  I think it behaves like t5 divebomb markers.

Reply:
Unlike twintania, Fan Ail always turns to where the divebomb marker appeared.

Reply:
Oh, you're totally right.  Thanks for letting me know.